src/components/ComponentSamples/ComponentSamplesPage.tsx

```diff
@@ -108,7 +108,7 @@ export const ComponentSamplesPage = () => {
     if (formState.component) {
       setSelected(formState.component);
     }
-  }, []);
+  }, [formState.component]);
```

Comment:
@rohitkrai03  can we remove this dependency. We just need setSelected called once when the component is mounted
```suggestion
  }, []);
```

Reply:
Done. Added a comment and disabled the eslint rule.